supportedBackends/aws/src/main/resources/sidecar/sidecar

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
```

Comment:
If you haven't already, we generally prefer that all shell scripts are run through [shellcheck](https://www.shellcheck.net/) and all warnings corrected

Reply:
Yup - I run with shellcheck. There was a missed warning from a last minute change that I've since cleaned up.